src/lib.rs

```diff
@@ -166,6 +166,25 @@ The minimum supported version of Rust for `glam` is `1.36.0`.
 */
 #![doc(html_root_url = "https://docs.rs/glam/0.10.2")]
 #![cfg_attr(not(feature = "std"), no_std)]
+#![cfg_attr(target_arch = "spirv", feature(register_attr, repr_simd))]
+
+#[cfg(all(target_arch = "spirv", feature = "std"))]
+compile_error!("`std` feature is not supported when building for SPIRV");
+
+#[cfg(all(target_arch = "spirv", feature = "glam-assert"))]
```

Comment:
I noticed in the Cargo.toml you had disabled the optional `bytemuck` dependency for `spirv`. Is it worth doing that for the rest?

Reply:
Ended up just removing this cfg out, since it didn't seem to be doing anything (was needed in an old draft, but not anymore)

In any case, I think this PR is now ready for review and merge consideration! :tada: